src/i18n/pt.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   hello: 'Olá',
   keywords: 'fkwl, from, kibera, with, love, quenia, volunteering, kids',
+  contactUs: 'Contact-nos',
```

Comment:
Typo on the Pt version.

```suggestion
  contactUs: 'Contacta-nos',
```

Reply:
:pray: